app/services/search_rdv_collectif.rb

```diff
@@ -22,6 +22,7 @@ def rdvs_collectif_at(motif, lieu, user = nil)
     def available_slots(motif, lieu)
       Rdv.collectif.future
         .with_remaining_seats
+        .where.not(status: "revoked")
```

Comment:
Faire un scope de ce where not le rendrait peut être plus explicite ?

Reply:
Tout à fait ! il existe déjà grace à la magie de rails et des enums en plus. `not_revoked`